frontend/src/views/IndividualStoryView.vue

```diff
@@ -18,6 +19,14 @@ onMounted(() => {
 	 })
 });
 
+// TODO: Look into alternate function to speed this up
```

Comment:
Is the slow part the loading of the spread sheet? We could cache the entire sheet's contents.

Reply:
I suspect it's the rendering instead of the loading, but will try out a cache to confirm. Going to look into https://router.vuejs.org/guide/advanced/data-fetching.html as well.